Sources/PackageCollections/Storage/SQLitePackageCollectionsStorage.swift

```diff
@@ -794,7 +794,7 @@ final class SQLitePackageCollectionsStorage: PackageCollectionsStorage, Closable
                     return false
                 }
             }
-        })
+        }
```

Comment:
this change intended ^^

Reply:
yes, it was a swiftformat change but I've reverted it in this PR.